ansible/server-state-playbooks/ome-dundeeomero/Vagrantfile

```diff
@@ -8,6 +8,7 @@ Vagrant.configure("2") do |config|
   end
 
   config.vm.provision "ansible" do |ansible|
+    ansible.skip_tags = "lvm"
```

Comment:
Since vagrant will be set up differently to the production systems, to let testing progress, skip `lvm`.